docs/installation/application.rst

```diff
@@ -41,40 +41,19 @@ install.yml
 
 This file is generated with your application scaffold. Please do NOT list any dependencies in setup.py. Instead list them in the :file:`install.yml` file. This file should be committed with your application code in order to aid installation on a Tethys Portal
 
-.. code-block:: python
-
-	# This file may be committed to your app code.
-	version: 1.0
-	# This should match the app - package name in your setup.py
-	name: hydroexplorer
-
-	requirements:
-	  # Putting in a skip true param will skip the entire section. Ignoring the option will assume it be set to False
-	  skip: false
-	  conda:
-        channels:
-          - conda-forge
-        packages:
-          - pyshp=2.0.0
-          - geojson
-          - shapely
-
-      pip:
-        - see
-
-	post:
-  	  - test.sh
+.. literalinclude:: resources/example-install.yml
```

Comment:
Can this file be downloaded?

Reply:
Not sure what you mean, but this shows the yml in a block like it would if the text was included in the rst, so it can be copied and pasted easily.